### ISSUE TITLE
Diagnostics "Path::Tiny paths require defined, positive-length parts"

### ISSUE THREAD

**Issue by andk** [robrwo/perl-Alien-Brotli#2]
Full fail report: http://www.cpantesters.org/cpan/report/e2b48d48-eeab-11ec-80bb-eaebc4fa55a7

With that perl I stepped through the test with the debugger:

``` text
  DB<0> f Alien/Brotli.pm
Choosing blib/lib/Alien/Brotli.pm matching 'Alien/Brotli.pm':

  DB<1> b 21
  DB<2> c
Alien::Brotli::exe(blib/lib/Alien/Brotli.pm:21):
21:         return path( $self->dist_dir, 'bin', $self->runtime_prop->{command} );
  DB<2> x $self->dist_dir
0  '/tmp/loop_over_bdir-bQg7EB/Alien-Brotli-v0.2.0-0/blib/lib/auto/share/dist/Alien-Brotli'
  DB<3> x $self->runtime_prop->{command}
0  undef
  DB<4> x $self->runtime_prop           
0  HASH(0x56194f00f2d0)
   'alien_build_version' => 2.48
   'alt' => HASH(0x56194edf2f50)
      'libbrotlicommon' => HASH(0x56194ef67720)
         'cflags' => ' '
         'cflags_static' => ' '
         'libs' => '-lbrotlicommon '
         'libs_static' => '-lbrotlicommon '
         'version' => '1.0.9'
      'libbrotlidec' => HASH(0x56194ef70118)
         'cflags' => ' '
         'cflags_static' => ' '
         'libs' => '-lbrotlidec '
         'libs_static' => '-lbrotlidec -lbrotlicommon '
         'version' => '1.0.9'
      'libbrotlienc' => HASH(0x56194ef701a8)
         'cflags' => ' '
         'cflags_static' => ' '
         'libs' => '-lbrotlienc '
         'libs_static' => '-lbrotlienc -lbrotlicommon '
         'version' => '1.0.9'
   'cflags' => ' '
   'cflags_static' => ' '
   'distdir' => '/tmp/loop_over_bdir-bQg7EB/Alien-Brotli-v0.2.0-0/blib/lib/auto/share/dist/Alien-Brotli'
   'install_type' => 'system'
   'legacy' => HASH(0x56194ef702c8)
      'finished_installing' => 1
      'install_type' => 'system'
      'name' => 'libbrotlicommon'
      'original_prefix' => '/home/sand/src/perl/repoperls/installed-perls/host/k93msid/v5.36.0/da1c/lib/site_perl/5.36.0/x86_64-linux/auto/share/dist/Alien-Brotli'
      'version' => '1.0.9'
   'libs' => '-lbrotlicommon '
   'libs_static' => '-lbrotlicommon '
   'perl_module_version' => 'v0.2.0'
   'prefix' => '/home/sand/src/perl/repoperls/installed-perls/host/k93msid/v5.36.0/da1c/lib/site_perl/5.36.0/x86_64-linux/auto/share/dist/Alien-Brotli'
   'version' => '1.0.9'                                                                                                                                                                              
```

**Comment by michal-josef-spacek** [comment]
This is because `$self->runtime_prop->{command}` returns undef in case of found brotli library via pkg-config.
We need to add another check for brotli command.

**Comment by michal-josef-spacek** [comment]
I don't know how to add `Alien::Build::Plugin::Probe::CommandLine` to cpanfile

**Comment by robrwo** [comment]
> I don't know how to add `Alien::Build::Plugin::Probe::CommandLine` to cpanfile

It should be added to the alienfile.

**Comment by robrwo** [comment]
> > I don't know how to add `Alien::Build::Plugin::Probe::CommandLine` to cpanfile
> 
> It should be added to the alienfile.

Actually, it's part of Alien::Build.

**Comment by michal-josef-spacek** [comment]
> > I don't know how to add `Alien::Build::Plugin::Probe::CommandLine` to cpanfile
> 
> It should be added to the alienfile.

In alienfile is plugin 'Probe::CommandLine' which means loading of this module.

**Comment by michal-josef-spacek** [comment]
> > > I don't know how to add `Alien::Build::Plugin::Probe::CommandLine` to cpanfile
> > 
> > 
> > It should be added to the alienfile.
> 
> Actually, it's part of Alien::Build.

Yes, still in cpanfile are files that are parts of Alien::Build.
BTW: I believe that is good practice to fill all required modules.

**Comment by michal-josef-spacek** [comment]
@robrwo I am looking to Alien::Build::Plugin::PkgConfig::Negotiate and there is code for adding itself to configure prereqs. Then this information is autogenerated to cpanfile.

Reason is different versions of Alien::Build::Plugin::PkgConfig::Negotiate for requires list :-)
https://github.com/PerlAlien/Alien-Build/blob/main/lib/Alien/Build/Plugin/PkgConfig/Negotiate.pm#L136

**Comment by michal-josef-spacek** [comment]
Fixed in 0.2.1
